juriscraper/opinions/united_states/territories/pr.py

```diff
@@ -6,48 +6,29 @@
 Court Short Name: Puerto Rico
 """
 
-import locale
-from datetime import date, datetime
-from typing import Optional
+from datetime import date
+
+from dateparser import parse
 
 from juriscraper.OpinionSiteLinear import OpinionSiteLinear
 
 
 class Site(OpinionSiteLinear):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        locale.setlocale(locale.LC_TIME, "es_ES.UTF-8")
         self.court_id = self.__module__
         year = date.today().year
+        year = 2022
         self.url = f"https://poderjudicial.pr/index.php/tribunal-supremo/decisiones-del-tribunal-supremo/decisiones-del-tribunal-supremo-{year}/"
```

Comment:
Well, this is a huge improvement over using `locale`, but do we want to hard-code the year? If so, should we put a comment explaining why?

Reply:
Ha no.  That was an accident.  They haven’t published any in 2023. So testing was done on 2022

Reply:
unwound, apologies again for it.